src/App.js

```diff
@@ -1,9 +1,12 @@
 import "./App.css";
 import "@google/model-viewer/dist/model-viewer.min.js";
 import ModelViewer from "./components/ModelViewer/ModelViewer";
+import Error404 from "./components/Error404";
 
 const App = () => {
+  if(window.location.pathname === "/")
   return <ModelViewer />;
+  else return <Error404/>;
 };
 
```

Comment:
The page looks good in terms of design but it will not work as a React app's page.
We need to use react-router and configure different pages to be shown on specific routes / URLs and if any of those URLs doesn't match then we show 404 page.

Take a look here : https://www.geeksforgeeks.org/how-to-setup-404-page-in-react-routing/